_posts/2023-08-23-critical-infrastructure-for-open-scholarship.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: post
+title: Critical infrastructure for Open Scholarship, and Policies to support wider participation in open data science
+authors:
+- yochannah
+- bethaniley
+image: https://images.unsplash.com/photo-1512314889357-e157c22f938d?
+photos:
+  name: Photo by AbsolutVision on Unsplash
+  url: https://unsplash.com/photos/82TpEld0_e4
+---
+
+
+This is a double announcement of two exciting (and linked) items: 
+
+1. OLS is delighted to welcome [FORRT](https://forrt.org/) as an organisational partner. 
+2. Thanks to the [Alan Turing Institute](https://www.turing.ac.uk/), we're working together towards policy briefings for widening participation in Open Data Science. 
+
+![A black and white logo of FORRT. The image comprises of a black fort/castle, with the word "FORRT" written in black letters next to it.](/images/2023-08-23-FORRT-logo.svg)
+
+## Why FORRT?
+The two parts of this story are directly linked. FORRT is a key player in **Open Scholarship**, with over 900 community members, an open science glossary, publications that explore open science and meta-science topics such as replications, curricular reform, and different ways to be inclusive to neurodiverse populations, but also an enormous people-powered array of educational resources democratizing access to learning opportunities and engagement in open scholarship. Explore their [website](https://forrt.org/) to see many of the amazing things they've been working on.
```

Comment:
@yochannah Can this sentence be broken in two? The length might make it a bit less comprehensible.

 E.g "_...but also an enormous..._" might begin a **new sentence**.

When you can, please have a look.

Reply:
```suggestion
The two parts of this story are directly linked. FORRT is a key player in **Open Scholarship**, with over 900 community members, an open science glossary, publications that explore open science and meta-science topics such as replications, curricular reform, and different ways to be inclusive to neurodiverse populations. They also have an enormous people-powered array of educational resources democratizing access to learning opportunities and engagement in open scholarship. Explore their [website](https://forrt.org/) to see many of the amazing things they've been working on.
```